src/PopEngine/index.ts

```diff
@@ -64,14 +64,23 @@ export class PopEngine {
     }
   }
 
+  public clearTimeoutByGroupId(groupId: string) {
+    this._maybeClearTimeout(this._timeouts.timeToHoverOnPop, groupId);
+    this._maybeClearTimeout(this._timeouts.hoverdelay, null);
+  }
+
+  public clearTimeout(targetElement: Element) {
+    let groupId = targetElement.getAttribute('popgun-group');
```

Comment:
how about doing what you did elsewhere and do

```javascript
let groupId = targetElement.getAttribute('popgun-group') || targetElement.getAttribute('pop-id');
```
and then making clearTimeoutByGroupId a private method?